ajax/libs/tempusdominus-bootstrap-4/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "license": "MIT",
+  "name": "tempusdominus-bootstrap-4",
+  "author": "Jonathan Peterson",
+  "description": "A date/time picker component designed to work with Bootstrap 4 and Momentjs. For usage, installation and demos see Project Site on GitHub",
+  "homepage": "https://tempusdominus.github.io/bootstrap-4/",
+  "keywords": [
+    "bootstrap",
+    "datepicker",
+    "datetimepicker",
+    "timepicker",
+    "moment"
+  ],
+  "filename": "tempusdominus-bootstrap-4.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tempusdominus/bootstrap-4.git"
+  },
+  "version": "5.0.0-alpha10",
+  "autoupdate": {
```

Comment:
How about using "npm" as auto-update source? I saw there are one more version there.
If you're alright with that. Here is auto-update config:
```js
  "npmName": "tempusdominus-bootstrap-4",
  "npmFileMap": [
    {
      "basePath": "build",
      "files": [
        "**/*"
      ]
    }
  ]
```

Or It would be great if you could provide releases currently only on npm on GitHub! In this case, auto-update config would be
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/tempusdominus/bootstrap-4.git",
    "fileMap": [
      {
        "basePath": "build",
        "files": [
          "**/*"
        ]
      }
    ]
  }
```